### DISCUSSION TITLE
Library apparently fails out-of-the-box if environment is not "local"

### DISCUSSION THREAD

**Discussion by Offlein** [dedoc/scramble#161] [Ideas]
This might just be a documentation request, but in our system we use the string "dev" as what Laravel defaults to calling "local".

This library states that as soon as it's installed, I should be able to access /docs/api and /docs/api.json, but I only got 403's out-of-the-box with those.

I saw that this library is described as somewhat early stage, so at first I was like, "Well it must be broken at the moment", but then I did a little more digging and found that it's got a hard-coded check in `RestrictedDocsAccess` for environment being `'local'`.

It'd be good to at least get a comment about that, since I almost uninstalled before even using it.

**Comment by romalytvynenko** [comment]
@Offlein Thanks for sharing. Will definitely add this to the docs, where I mention the routes. 

Let me know if you had any other issues when starting with Scramble.

**Reply by Offlein** [reply]
Thanks @romalytvynenko!

After this I actually paused again -- I'm using spatie/laravel-data for DTOs and Resources, and that handles Validation and stuff currently, so it looked as if I might have to do a lot to get Scramble working in our system. :( 

I had a weird experience where, when I COULD access /docs/api I got a message about being unable to access the server (or something?) but I was also clearly accessing the endpoint. /docs/api.json DID seem to work. My guess was that the former was related to us using Docker maybe?

**Reply by romalytvynenko** [reply]
@Offlein, I'm planning to add `spatie/laravel-data` support at some point!

Regarding that experience of endpoint being not available, I'd need more info. Maybe some screenshots of responses from every endpoint, and screenshot of "Network" tab when `/docs/api` is opened. Not critical if you decided to pause on Scramble.

Thanks!

**Reply by Offlein** [reply]
@romalytvynenko Oh that's great. I'd be super willing to check it out. (Or possibly even help/do it? I can't tell how straightforward it would be though?)

I am currently doing other stuff in our internal project, but I do intend to come back to our internal API documentation and would try to get the details of the HTML error and Network response[s] then!
